packages/components/src/DataList/components/DataListOverflowFade/DataListOverflowFade.css

```diff
@@ -1,5 +1,5 @@
 .fadeContainer {
-  --overflow-fade--offset: 0;
+  --overflow-fade--offset: var(--space-smaller);
```

Comment:
We're going to need to bring back the "negative offset" from before:
![image](https://github.com/user-attachments/assets/cd91e8bc-bb54-4ac2-b15c-72eff2a035c4)

(I got the top screenshot by going back to a branch from 2 weeks ago)

Reply:
Yup just pushed that 😅